src/Symfony/Component/HttpFoundation/Tests/ResponseTest.php

```diff
@@ -206,6 +206,38 @@ public function testIsNotModifiedEtag()
 
         $response->headers->set('ETag', '');
         $this->assertFalse($response->isNotModified($request));
+
+        // Test wildcard
+        $request = new Request();
+        $request->headers->set('If-None-Match', '*');
+
+        $response->headers->set('ETag', $etagOne);
+        $this->assertTrue($response->isNotModified($request));
+    }
+
+    public function testIsNotModifiedWeakEtag()
+    {
+        $etag = 'randomly_generated_etag';
+        $weakEtag = 'W/randomly_generated_etag';
```

Comment:
This test is broken! It should have been quoted, meaning it does not compare etags correctly.

Reply:
Thank you for noticing. 
Could you submit a PR fixing this with a small explanation?

Reply:
Sure, I already wrote code that normalizes the etags before comparison and have unit tests running for them to match the RFC, so it should be perfectly doable. This was noticed after a few _days_ of pulling hairs out of my head wondering why I so seldom got a HTTP 304, ref https://stackoverflow.com/a/70670045/200987.